logbook-core/src/main/java/org/zalando/logbook/ChunkingHttpLogWriter.java

```diff
@@ -0,0 +1,42 @@
+package org.zalando.logbook;
+
+import org.zalando.logbook.DefaultLogbook.SimpleCorrelation;
+import org.zalando.logbook.DefaultLogbook.SimplePrecorrelation;
+
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+public final class ChunkingHttpLogWriter implements HttpLogWriter {
+
+    private final Pattern pattern;
+    private final HttpLogWriter writer;
+
+    public ChunkingHttpLogWriter(final int size, final HttpLogWriter writer) {
+        this.pattern = Pattern.compile("(?<=\\G.{" + size + "})");
```

Comment:
This looks like a very inefficient way to split a string into fixed length subparts. A better alternative might be to use a `StringReader`.